src/Adapters/Logger/WinstonLoggerAdapter.js

```diff
@@ -1,5 +1,5 @@
 import { LoggerAdapter } from './LoggerAdapter';
-import { logger, addTransport } from '../../logger';
+import { logger, addTransport, configureLogger } from './WinstonLogger';
```

Comment:
Could we just move WinstonLogger functionality inside this class? May simplify things.
The main problem I see if we did that is creating the default logger in logging.js - one solution would be to create a WinstonLoggerAdapter in logging.js and call get logger() on that, instead of creating a WinstonLogger instance.